systemd/cloud-config.target

```diff
@@ -14,5 +14,5 @@
 
 [Unit]
 Description=Cloud-config availability
-Wants=cloud-init-local.service cloud-init.service
-After=cloud-init-local.service cloud-init.service
+Wants=cloud-init-local.service cloud-init-network.service
+After=cloud-init-local.service cloud-init-network.service
```

Comment:
There is a debian patch in `debian/patches/no-single-process.patch` which disables the single-process for the SRU. That patch also removes the `cloud-init-network.service.tmpl` file, so I'm thinking this rename here should be reverted in that patch too?